### PR TITLE
have a skip-ifup option

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

File: snappy/oem.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ type OEM struct {
 		BootAssets *BootAssets      `yaml:"boot-assets,omitempty"`
 	} `yaml:"hardware,omitempty"`
 	Software Software `yaml:"software,omitempty"`
+	//
```
Comment (mvo5) [inline snappy/oem.go]: This comment looks a bit empty :)

**Comment by mvo5** [comment]
Looks great, just one small comment.

**Comment by elopio** [comment]
Do you have an example of a snap that ifup-s the interface?

File: snappy/oem.go (hunk 1 above)
Comment (chipaca) [inline snappy/oem.go]: What part of “” do you not understand?

**Comment by chipaca** [comment]
@elopio all snaps should ifup the interface *unless* the oem package.yaml has
```yaml
    oem:
        skip-ifup-provisioning: true
```

**Comment by chipaca** [comment]
@elopio I downloaded lp:~snappy-dev/snappy-hub/snappy-systems, added that stanza to the oem.config, created an image with that, booted it, ssh'd in (so, it had network on first boot), updated the snappy, removed the firstboot and ens3 files, rebooted, and got no ethernet \o/

The resulting package.yaml for the oem snap was, in total:

```yaml
name: generic-amd64
vendor: Sergio Schvezov <sergio.schvezov@canonical.com>
icon: meta/ubuntu.png
version: 1.4
type: oem

config:
    ubuntu-core:
        autopilot: true

oem:
    branding:
        name:  amd64
        subname: generic

    skip-ifup-provisioning: true

    hardware:
        bootloader: grub
        architecture: amd64
        partition-layout: system-AB
        boot-assets:
            files:
                - path: grub.cfg
```

**Comment by elopio** [comment]
I want to state and record my happiness about this branch. :+1:

**Pull request merged**